src/Console/Command/Cache.php

```diff
@@ -65,9 +65,14 @@ public function handle()
         }
 
         $stub = "app('api.router')->setAdapterRoutes(unserialize(base64_decode('{{routes}}')));";
+        $path = $this->laravel->getCachedRoutesPath();
+
+        if (! $this->files->exists($path)) {
+            $stub = "<?php\n\n".$stub;
```

Comment:
Yeah... that might be better, don't know why it didn't occur to me. Going to fix it now.